### ISSUE TITLE
Why error occurs but programme is not affacted.

### ISSUE THREAD

**Issue by zoulei** [tensorflow/tensorflow#35830]
I use python to train a model and export the model. Then I use c++ to import the exported model and use it to predict data. However, during the prediction process, error message is printed during the screen, but the programme run normally with no exception. Below is the error message.

`2020-01-14 03:29:49.794822: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
2020-01-14 03:29:51.689021: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
2020-01-14 03:29:51.689069: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
2020-01-14 03:29:51.702364: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
2020-01-14 03:29:53.728503: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
2020-01-14 03:29:55.683826: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
2020-01-14 03:29:55.686735: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
2020-01-14 03:29:55.686879: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
2020-01-14 03:29:55.686918: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
2020-01-14 03:29:55.687032: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
2020-01-14 03:29:55.687120: W tensorflow/core/framework/op_kernel.cc:1401] OP_REQUIRES failed at example_parsing_ops.cc:144 : Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]
Error running session: Invalid argument: Name: <unknown>, Feature: 100 (data type: float) is required but could not be found.
         [[{{node ParseExample/ParseExample}}]]`

Below is the code that use the model in c++.
`float NodeEVEngine::getmlev(int curturn,vector<float> &inputfeature,bool allin,bool turnover){
//    MyTimer::GetInstance()->start("prepare");
    Example example;
    Features features;
    int featurelen = inputfeature.size();
    for (int i = 0; i < featurelen; i++) {
        FloatList floatList;
        floatList.add_value(inputfeature[i]);
        Feature feature;
        *feature.mutable_float_list() = floatList;
        features.mutable_feature()->operator[](to_string(i)) = feature;
    }
    *example.mutable_features()=features;
    std::string test_str;
    example.SerializeToString(&test_str);
    tensorflow::Input input({test_str});
    vector<string> outputname({"dnn/logits/BiasAdd:0"});
    std::vector<Tensor> outputs;
    Status run_status;
//    MyTimer::GetInstance()->stopandprintandclear("prepare");
//    MyTimer::GetInstance()->start("mlev");
    if (allin)run_status = m_allinbundle[curturn-PREFLOPNO]->session->Run({{"input_example_tensor",input.tensor()}}, outputname, {}, &outputs);
    else if (turnover)run_status = m_turnoverbundle[curturn-PREFLOPNO]->session->Run({{"input_example_tensor",input.tensor()}}, outputname, {}, &outputs);
    else run_status = m_bundle[curturn-PREFLOPNO]->session->Run({{"input_example_tensor",input.tensor()}}, outputname, {}, &outputs);
    if (!run_status.ok()) {
        cout << "Error running session: " << run_status << std::endl;
        return -1;
    }
//    MyTimer::GetInstance()->stopandprintandclear("mlev");
//    PrintVector(inputfeature,"inputfeature:");
    return outputs[0].tensor<float,2>()(0,0);
}`
